test/test-applications/security/aspnet/Samples.AspNetMvc5/Content/bootstrap.css

```diff
@@ -0,0 +1,6834 @@
+/*!
```

Comment:
All these bootstap, jquery, fonts etc adds a lot of bloat. Can we remove them and reference the existing resources (or just don't include them at all)? Same for the aspnetcore sample

Reply:
Yes, done in 7ef0c718 and baac756f